src/apprt/gtk/App.zig

```diff
@@ -44,6 +44,9 @@ const log = std.log.scoped(.gtk);
 
 pub const Options = struct {};
 
+/// A global variable indicating if user has dismissed the notice about existing crash reports
+pub var crash_reports_notice_dismissed = false;
+
```

Comment:
Global states are a bit yikes and we should avoid them. I think setting this as a field of GTK App works just fine.